src/content/release/breaking-changes/android-surface-plugins.md

```diff
@@ -75,6 +75,27 @@ surfaceProducer.setCallback(
 A full example of using this new API can be found in [PR 6989][] for the
 `video_player_android` plugin.
 
+## Note on camera previews
+
+If your plugin implements a camera preview, your migration may also require
+fixing the rotation of that preview. This is because `Surface`s produced by the
+`SurfaceProducer` may not contain the transformation information that Android
+libraries need to correctly rotate the preview automatically.
+
+In order to correct the rotation, you will need to rotate the preview with
+respect to the camera sensor orientation and the device orientation according
+to the equation:
+
+```plaintext
+rotation = (sensorOrientationDegrees - deviceOrientationDegrees * sign + 360) % 360
```

Comment:
Should we provide more context on how to retrieve `sensorOrientationDegrees` and `deviceOrientationDegrees`
directly in this document? (instead of pointing people to a GitHub PR)

We provide code samples for other things in this migration guide (see line 59 in this file), so can we do the same for this addition as well? I don't feel string about this, but maybe not everyone writing plugins / reading this doc is okay with going through a potentially long Github PR to find what they need, to migrate.

I would definitely keep the PR link, though, so people have a completed migration to look at as well.

Reply:
I added some links for further clarification because I agree that the PR isn't enough. We also had to make some Flutter camera plugin-specific fixes in it, so I don't want the focus to be there.

I think I'll leave it here unless others feel strongly because I think camera plugin authors are familiar with the APIs and could make the migration with this information.